media/css/mozorg/credits.scss

```
@@ -3,13 +3,20 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 html {
-    background: -moz-dialog;
-    color: -moz-dialogtext;
+    background: #ffffff;
```

Linting check failed because this can be shortened
```suggestion
    background: #fff;
```

---

Fixed.